### PR TITLE
Use updated json

### PR DESCRIPTION
**Pull request by minaorangina**
Completes the next step to update the website with upcoming event deets :) Also cleaned up the index page a bit, lots of redundant links and stuff there.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -8,8 +8,11 @@
   <!-- CSS  -->
   <link href="css/materialize.css" type="text/css" rel="stylesheet" media="screen,projection"/>
   <link href="css/style.css" type="text/css" rel="stylesheet" media="screen,projection"/>
-  <link href="css/team-images.css" type="text/css" rel="stylesheet" media="screen,projection"/>
+  <!-- <link href="css/team-images.css" type="text/css" rel="stylesheet" media="screen,projection"/> -->
   <link href="http://maxcdn.bootstrapcdn.com/font-awesome/4.1.0/css/font-awesome.min.css" rel="stylesheet">
+  <script src="https://cdn.jsdelivr.net/npm/es6-promise@4/dist/es6-promise.auto.min.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/fetch/2.0.3/fetch.min.js"></script>
+  <script src="js/moment.min.js"></script>
```
Comment (msmichellegar) [inline index.html]: Do these scripts need to be at the bottom of the html file?

**Review by msmichellegar — COMMENTED** [review]
Awesome, so the homepage is also now dynamically generated? This is SUCH an improvement :)

Just had one question about the placement of the scripts.

File: index.html [hunk 2]
```diff
@@ -8,8 +8,11 @@
   <!-- CSS  -->
   <link href="css/materialize.css" type="text/css" rel="stylesheet" media="screen,projection"/>
   <link href="css/style.css" type="text/css" rel="stylesheet" media="screen,projection"/>
-  <link href="css/team-images.css" type="text/css" rel="stylesheet" media="screen,projection"/>
+  <!-- <link href="css/team-images.css" type="text/css" rel="stylesheet" media="screen,projection"/> -->
```
Comment (claireinez) [inline index.html]: Why not just delete this link if it's not needed?

File: js/index-template.js [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+// PRODUCTION URL
+// prodUrl requires the latest commit hash for the `json` branch (the merge commit from the last PR)
+// Can be the short or long version.
+var commitHash = "04a6741";
+var prodUrl = "https://cdn.rawgit.com/node-girls/node-girls-website/" + commitHash + "/events.json";
+
+// DEVELOPMENT URL
+// To see if your updated events.json is working, set this variable to the name
+// of the branch you're working from (after you've pushed to GitHub)
+var yourWorkingBranchName = "";
```
Comment (claireinez) [inline js/index-template.js:14]: Minor niggle but I feel that lines 14 and 15 should perhaps be in a comment as they're not actually used in production but I'm not bothered if they are kept in.

**Review by claireinez — APPROVED** [review]
I only have minor points that won't block merging.

**Review by msmichellegar — APPROVED** [review]

**Pull request merged**